docs/health/index.rst

```diff
@@ -2,62 +2,137 @@
+By using pyATS Health Check, it is possible to check the CPU load, memory usage, detection of specific log messages, and whether or not the core file created in the event of a crash or malfunction is generated.
 
-The health check is driven by a `health_file` which is provided at run time. There is two different mechanism to run the checks:
+pyATS Health Check currently supports the following 4 checks by default.
 
-1. Part of a pre/post processor of a section or testcase.
-2. Part of another process continuously (Coming up in September).
+.. list-table::
+   :header-rows: 1
 
-pyATS Health Check leverages existing :ref:`Blitz<blitz>` style YAML format and use same format and capabilities what Blitz has.
-The actions in pyATS Health Check yaml, those actions will be added to pyATS job's each section as pre/post processors. Pre, Post or both processors can be specified in the yaml.
+   * - health check
+     - description
+   * - cpu
+     - cpu load check. check total of cpu load with threshold 90% by default.
+   * - memory
+     - memory usage check. check total of memory usage with threshold 90% by default.
+   * - logging
+     - keyword check in show logging output. default keywords are traceback, Traceback, TRACEBACK
+   * - core
+     - check if core file is generated on device. just check by default. Please use `--health-remote-device` to copy the core file to remote server.
 
-.. note::
 
-     See the detail of :ref:`Blitz<blitz>`
+Using pyATS Health Check is very easy. Just list the above check names with `--health-checks` in the current pyats command.
 
 .. code-block:: bash
 
-    # Quick start Guide!
-    # ------------------
-    # 1. Create the health yaml
-    # 2. Add the arguments
-    # 3. (Optional) Run it and hope no issue is found by the pyATS Health Check!
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core
+
+If you want to check only cpu, please specify just only `cpu` to `--health-checks`. **Also, importantly, the pyATS Health Check by `--health-checks` runs as a post-processor after each test case.**
+
+The core file is only detection by default, but if you want to send it to TAC for analysis or move it to the remote server at the time of detection, you can enumerate the remote server information with `--health-remote-device` to copy the generated core file. It is possible to copy the file at the time of detection and delete the file from the device.
+
+The command example in that case is as follows.
+
+.. code-block:: bash
 
-    # Integrated with pyATS jobs
-    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-file /path/to/health.yaml
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-remote-device name:myserver path:/tmp/ protocol:http --health-mgmt-vrf iosxe:None
```

Comment:
health-remote-device should be a url?

Reply:
why? it's sever name under `server` block in testbed yaml.

Reply:
You are not just specifying the server, you are adding path and protocol. URL is the common syntax to use for this kind of thing. Using an argument `--health-upload <path|url>` seems like a better syntax? You can use urlparse/FileUtils.parse_url like in filetransferutils to handle this.

Reply:
this is not only for http. for tftp/ftp/scp/http.

Reply:
yes and still url syntax applies to all of those...

Reply:
I understand what you mean. but the idea was not during review. if really need, we can switch to new argument in the future.